content/en/docs/instrumentation/php/logging.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Logging
+weight: 10
+---
+
+## Introduction
+
+As logging is a mature and well-established function, the
```

Comment:
Not sure that "function" is the right word here.

Reply:
"process" ?

Reply:
programming tool? (or just "tool")

Can't think of better word(s), and I don't want this to be blocking. So I'll let you decide, and then mark this as resolved so we can move on.

Reply:
Would not "feature" be right word here?

Reply:
I would vote for "practice", as it reads like a verb to me.

...but that's now introducing a _fifth_ word into the mix 😆